src/services/infrastructure/url-generator/url.generator.service.ts

```diff
@@ -540,72 +579,70 @@ export class UrlGeneratorService {
       calloutId: string;
     }[] = await this.entityManager.connection.query(
       `
-        SELECT callout_contribution.id as contributionId, callout_contribution.calloutId as calloutId FROM callout_contribution
+        SELECT callout_contribution.id, callout_contribution.calloutId as calloutId FROM callout_contribution
```

Comment:
It doesn't seem we use the `callout_contribution.id`
```suggestion
        SELECT callout_contribution.calloutId as calloutId FROM callout_contribution
```